app.js

```diff
@@ -139,6 +126,14 @@ function handleClick(event){
     displayImageOne.removeEventListener('click', handleClick);
     displayImageTwo.removeEventListener('click', handleClick);
     displayImageThree.removeEventListener('click', handleClick);
+
+    function saveProductsToLocalStorage(allProducts){
```

Comment:
you should declare this function outside of your click handler in case you need to use it elsewhere in your program